tests/kernel/run_ltp.pm

```diff
@@ -337,187 +337,56 @@ sub run {
     my ($self) = @_;
     my $cmd_file = get_var 'LTP_COMMAND_FILE';
     die 'Need LTP_COMMAND_FILE to know which tests to run' unless $cmd_file;
-    my $cmd_pattern = get_var('LTP_COMMAND_PATTERN') || '.*';
-    my $cmd_exclude = get_var('LTP_COMMAND_EXCLUDE') || '$^';
     my $timeout     = get_var('LTP_TIMEOUT')         || 900;
-    my $ltp_env     = get_var('LTP_ENV');
     my $is_posix    = $cmd_file =~ m/^\s*openposix\s*$/i;
-    my $is_network  = $cmd_file =~ m/^\s*(net|net_stress)\./;
     my $tmp;
 
-    if ($ltp_env) {
-        $ltp_env =~ s/,/ /g;
-        script_run("export $ltp_env");
-        script_run('env');
-    }
-
-    my $test_result_export = {
-        format  => 'result_array:v1',
-        results => []};
+    # my $test_result_export = {
+    #     format  => 'result_array:v1',
+    #     results => []};
```

Comment:
Don't add disabled code as nobody but you will understand why this is not active. Better leave it out and keep in your local git repository, either in `git stash` or a temporary "WIP"-commit